src/core/sagas/Sagas.js

```diff
@@ -29,5 +30,9 @@ export default function* sagas() :Saga<*> {
     // RoutingSagas
     fork(RoutingSagas.goToRootWatcher),
     fork(RoutingSagas.goToRouteWatcher),
+
+    fork(AcccessSagas.getAllAccessRequestsWatcher),
+    fork(AcccessSagas.submitAccessRequestWatcher),
+    fork(AcccessSagas.updateAccessRequestWatcher),
```

Comment:
It makes sense for this to exist for storybook, but as an embedded module, this file won't actually be needed in the bundle, right? We would add these lines to `Sagas.js` in the parent app, right?

Reply:
The module handles itself when the top level components are mounted. Using the components should not incur any additional "glue code". The redux store for the module is completely separate and needs watch for and respond to its own actions.